react/IntentIframe/IntentIframe.jsx

```diff
@@ -17,7 +19,13 @@ class IntentIframe extends React.Component {
   componentDidMount() {
     const { action, data, type, onCancel, onError, onTerminate } = this.props
 
-    const create = this.props.create || cozy.client.intents.create
+    let create
+    if (this.props.create) {
+      create = this.props.create
+    } else {
+      const intents = new Intents({ client: this.props.client })
```

Comment:
If `client` is in DidMount props, maybe we should write `new Intents({ client })` ?